app/src/main/java/com/twilio/video/app/ui/room/RoomActivity.java

```diff
@@ -1155,6 +1158,14 @@ private Completable updateEnv() {
                                     RoomActivity.this,
                                     connectOptionsBuilder.build(),
                                     roomListener());
+
+                    Intent serviceIntent = VideoService.Companion.createIntent(this);
```

Comment:
Sorry, what I meant was to move the startService logic to `VideoService`

so like something like

```
VideoService.Companion.startForeground(this);
```